integration_tests/e2e/confirmCellMove.cy.ts

```diff
@@ -46,6 +49,7 @@ context('A user can confirm the cell move', () => {
         description: 'Behaviour',
       },
     ])
+    cy.task('stubOffenderBasicDetails', offenderBasicDetails)
```

Comment:
This was needed to allow the confirmation page to be shown (otherwise get a 404 though the existing tests still passed)